domains/Accounts/Resources/UserResource.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Domains\Accounts\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class UserResource extends JsonResource
+{
+    public function toArray($request): array
+    {
+        return [
+            'id'         => $this->id,
+            'name'       => $this->name,
+            'email'      => $this->email,
+            'created_at' => $this->created_at,
+            'updated_at' => $this->updated_at,
+        ];
```

Comment:
We could also expose the `trusted` and the `deleted_at` properties, too.

Reply:
I'm sorry but I don't understand.

My thought was, the "delete_at" field is unnecessary because if it is filled  then the user should not see their accounts.

Reply:
That's because you're only considering the `UserResource` useful only for being returned when requested via the profile route. However, we can use that same resource class in conjunction with a Link resource, or the new Question resource. Basically wherever it needs to be shown with a relatable object.

Reply:
Okay, I add. I don't normally show the "delete_at" field to end users.